Content/startbootstrap-clean-blog-gh-pages/gulpfile.js

```diff
@@ -0,0 +1,116 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var header = require('gulp-header');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/celestin211/celestinBlog/pullRequest?prid=8839082)
```suggestion
var header = require("gulp-header");
```